### PR TITLE
Updated to dotnet 48

### PR DESCRIPTION
**Pull request by asos-davemills**

### CODE REVIEW

File: src/Sitecore.Ship.Infrastructure/Helpers/Utilities.cs [hunk 1]
```diff
@@ -36,39 +33,39 @@ public static string SitecoreTempDirectory
...
             string fileName = Path.GetFileName(filePathToUnzipTo);
 
-            targetFile.ExtractItem(pathToUnzipTo, false, fileName);
+            targetFile.ExtractItem(reader, pathToUnzipTo, false, fileName);
             return true;
         }
 
         
-        public static void ExtractAll(string zipFile, string pathTo, bool includeTargetPathing = true)
+        public static void ExtractAll(string zipFileName, string pathTo, bool includeTargetPathing = true)
```
Comment (asos-piotrmajor) [inline src/Sitecore.Ship.Infrastructure/Helpers/Utilities.cs]: method never used

File: tests/intg-test/Sitecore.Ship.Infrastructure.Intg.Test/MockZipUtils.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+using ICSharpCode.SharpZipLib.Zip;
+using Sitecore.IO;
+
+namespace Sitecore.Ship.Infrastructure.Intg.Test
+{
+  public class MockZipUtils : BaseZipUtil
```
Comment (asos-piotrmajor) [inline tests/intg-test/Sitecore.Ship.Infrastructure.Intg.Test/MockZipUtils.cs]: class never used

File: src/Sitecore.Ship.Infrastructure/Install/PackageManifestReader.cs [hunk 3]
```diff
@@ -1,63 +1,85 @@
...
 using Sitecore.Install.Zip;
+using Sitecore.IO;
 using Sitecore.Ship.Core.Contracts;
 using Sitecore.Ship.Core.Domain;
 using Sitecore.Ship.Core.Services;
-using Sitecore.Zip;
 
 namespace Sitecore.Ship.Infrastructure.Install
 {
-    public class PackageManifestReader : IPackageManifestRepository
```
Comment (asos-piotrmajor) [inline src/Sitecore.Ship.Infrastructure/Install/PackageManifestReader.cs:13]: file needs formatting

File: tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <package id="Castle.Core" version="3.3.3" targetFramework="net45" />
```
Comment (asos-piotrmajor) [inline tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config:3]: do we need a dependency to Castle.Core 3.3.3.?

File: src/Sitecore.Ship.AspNet/Sitecore.Ship.AspNet.csproj [hunk 5]
```diff
@@ -34,14 +34,32 @@
...
-    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Newtonsoft.Json.11.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Microsoft.Extensions.DependencyInjection, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Microsoft.Extensions.DependencyInjection.6.0.0\lib\net461\Microsoft.Extensions.DependencyInjection.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Newtonsoft.Json.13.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="Sitecore.Kernel">
+      <HintPath>..\..\packages\ASOS.Content.Sitecore.Binaries.10.3.0.208-03vuln\lib\net48\Sitecore.Kernel.dll</HintPath>
```
Comment (asos-piotrmajor) [inline src/Sitecore.Ship.AspNet/Sitecore.Ship.AspNet.csproj]: I'm thinking that Vanilla has been merged to master, so we can include a stable package

File: tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config (hunk 4 above)
Comment (asos-deb) [reply tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config:3]: I haven't made any change to the Castle.Core libraries as it was not required to change or update.

File: src/Sitecore.Ship.Infrastructure/Helpers/Utilities.cs (hunk 1 above)
Comment (asos-deb) [reply src/Sitecore.Ship.Infrastructure/Helpers/Utilities.cs]: removed

File: tests/intg-test/Sitecore.Ship.Infrastructure.Intg.Test/MockZipUtils.cs (hunk 2 above)
Comment (asos-deb) [reply tests/intg-test/Sitecore.Ship.Infrastructure.Intg.Test/MockZipUtils.cs]: removed

File: src/Sitecore.Ship.Infrastructure/Install/PackageManifestReader.cs (hunk 3 above)
Comment (asos-deb) [reply src/Sitecore.Ship.Infrastructure/Install/PackageManifestReader.cs:13]: formatted

File: src/Sitecore.Ship.AspNet/Sitecore.Ship.AspNet.csproj (hunk 5 above)
Comment (asos-deb) [reply src/Sitecore.Ship.AspNet/Sitecore.Ship.AspNet.csproj]: updated to the latest release version

File: tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config (hunk 4 above)
Comment (asos-piotrmajor) [reply tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config:3]: 👍

**Review by asos-piotrmajor — APPROVED** [review]

File: tests/acceptance-test/Sitecore.Ship.AspNet.Web.Accept.Test/Sitecore.Ship.AspNet.Web.Accept.Test.csproj [hunk 6]
```diff
@@ -14,6 +14,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Sitecore.Ship.AspNet.Web.Accept.Test</RootNamespace>
     <AssemblyName>Sitecore.Ship.AspNet.Web.Accept.Test</AssemblyName>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
     <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (asos-markkidd) [inline tests/acceptance-test/Sitecore.Ship.AspNet.Web.Accept.Test/Sitecore.Ship.AspNet.Web.Accept.Test.csproj]: surely this needs removing what happens after we target 4.8 then 4.7.2, does it go back to 4.7.2 as in the ide I get awarning that the ref'd AspNet project targets an higher version ?

File: src/Sitecore.Ship.AspNet/Sitecore.Ship.AspNet.csproj (hunk 5 above)
Comment (asos-markkidd) [reply src/Sitecore.Ship.AspNet/Sitecore.Ship.AspNet.csproj]: We shouldn't be referencing Sitecore assemblies from here, read the Readme.md in libs src/directory.

**Review by asos-markkidd — APPROVED** [review]

**Pull request merged**